plugins/services/src/js/components/forms/HealthChecksFormSection.js

```diff
@@ -13,89 +14,99 @@ import FormGroupContainer from '../../../../../../src/js/components/form/FormGro
 
 import {FormReducer as healthChecks} from '../../reducers/serviceForm/HealthChecks';
 
+const errorsLens = Objektiv.attr('healthChecks', []);
+
 class HealthChecksFormSection extends Component {
+  getErrors(key) {
+    return errorsLens.at(key, {}).get(this.props.errors);
+  }
+
   getAdvancedSettings(healthCheck, key) {
-    const {healthChecks} = this.props.errors;
     if (healthCheck.protocol !== 'COMMAND' && healthCheck.protocol !== 'HTTP' &&
       healthCheck.protocol !== 'HTTPS') {
       return null;
     }
 
+    const errors = this.getErrors(key);
```

Comment:
I mean, it's more complex. I use default value for every lens so that we can go deeper even when we don't have the full path in `errors`